patches/mcp-session-status-fix.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/src/mcp/server/sse.py b/src/mcp/server/sse.py
```

Comment:
This probably not intended to be merged this way

Reply:
I didn't know how it's supposed to work tbh. I just kinda modeles it based on the rest of the patches.

Reply:
we decided on standup to postpone this